docs/extending/customizing_richtext_output.md

```diff
@@ -0,0 +1,62 @@
+(customizing_richtext_output)=
+
+# Customizing richtext output
+In some cases, it might be necessary to customize the output of a [RichTextField](rich_text_field), or a `RichTextBlock` within a [StreamField](../topics/streamfield).
```

Comment:
```suggestion
In some cases, it might be necessary to customize the output of a [RichTextField](rich_text_field), or a `RichTextBlock` within a [StreamField](streamfield_topic).
```